### ISSUE TITLE
IMDaemonCore/IMDChat.h file not found

### ISSUE THREAD

**Issue by mitchell183** [hbang/TypeStatus#79]
Hi,
getting this error when compiling. 

```
==> Compiling ReadReceiptMirror.x (armv7)…
ReadReceiptMirror.x:2:9: fatal error: 'IMDaemonCore/IMDChat.h' file not found
#import <IMDaemonCore/IMDChat.h>
        ^
1 error generated.

```

Tried compiling for 9.2, 9.3 and 10.1 with the patched sdks.

https://github.com/theos/sdks

Any idea?

**Comment by kirb** [comment]
I hadn’t pushed these headers yet; will do later.

**Comment by kirb** [comment]
Fixed a while ago in theos/headers@a2f2e4d.
